pkgs/build-support/dlang/builddubpackage/hooks/dub-build-hook.sh

```diff
@@ -0,0 +1,13 @@
+dubBuildHook() {
+    runHook preBuild
+    echo "Executing dubBuildHook"
```

Comment:
use nixLog* functions.

Reply:
Could you send an example?, as I don't seem to be able to find what you're referencing here

Reply:
`nix*Log` famili is a more or less recent addition to nixpkgs.

Shameless auto-promotion: https://github.com/NixOS/nixpkgs/pull/338789

Reply:
Sorry for the slow response.

I'm not able to see any of the output from `nix*Log` functions.
I tried `NIX_DEBUG=7 nix-build -A some-package --verbose --impure`
It seems that `NIX_DEBUG` is not set at all for me during the build.

Is there something I'm missing?

Reply:
No, there is nothing that you're missing.

Setting NIX_DEBUG is an ordeal. You have to recompile Nix itself. Here's a hint.

- https://github.com/NixOS/nixpkgs/pull/331794

Reply:
Hmm, I see, thanks for the info.
Seeing this, I don't think I really want to use the nix*Log functions here, so I'll stay with the status-quo.
If it gets replaced anyway in a treewide, I wont complain.